### DISCUSSION TITLE
Remove quest attunement to queue in normal OHF/BM

### DISCUSSION THREAD

**Discussion by bigdeekpumper** [chromiecraft/chromiecraft#7002] [Suggestions]
Since the attunement requirements for Karazhan and SSC have been removed, I propose that the quest requirement to queue up OHF/BM normal should be removed too. This would provide leveling players with more variety during RDF spam (OHF appears as level 64 on the RDF list, while the quest requires level 66) and will save them valuable time (OHF queue time can be huge sometimes). 

(I'm not even sure this "attunement" was required in original WotLK RDF but I couldn't find any source besides my memory. )

**Comment by heyitsbench** [comment]
The reason the attunements for Karazhan/SSC have been removed is because they are not supposed to be active as of 3.3.5a. If you can find evidence that WotLK is supposed to have OHF/BM unobstructed in RDF, then that's the only scenario I can imagine the restriction being lifted.
